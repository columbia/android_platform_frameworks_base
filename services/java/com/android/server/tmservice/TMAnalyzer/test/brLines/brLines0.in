E/dalvikvmtm(  592): 63|1 |001a|Ledu/columbia/gpsapp/MainActivity;.sendMessage|| if-gez| > 
E/dalvikvmtm(  592): 64|1 |0023|Ledu/columbia/gpsapp/MainActivity;.sendMessage|| if-nez| > 
E/dalvikvmtm(  592): 65|1 |00b5|Ledu/columbia/gpsapp/MainActivity;.sendMessage|| if-nez| < 
E/dalvikvmtm(  592): 66|1 |0007|Ledu/columbia/gpsapp/MainActivity$Msg;.toString|| if-nez| > 
E/dalvikvmtm(  592): 67|1 |013d|Ledu/columbia/gpsapp/MainActivity$Msg;.toString|| if-ne| < 
E/dalvikvmtm(  592): 68|1 |0155|Ledu/columbia/gpsapp/MainActivity$Msg;.toString|| if-nez| < 
E/dalvikvmtm(  592): 69|1 |015f|Ledu/columbia/gpsapp/MainActivity$Msg;.toString|| if-nez| < 
