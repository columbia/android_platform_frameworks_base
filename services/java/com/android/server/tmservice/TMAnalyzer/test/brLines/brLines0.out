BR::592 (0) :: 0 :: Ledu/columbia/gpsapp/MainActivity .sendMessage @ 26 :: True
BR::592 (0) :: 0 :: Ledu/columbia/gpsapp/MainActivity .sendMessage @ 35 :: True
BR::592 (0) :: 0 :: Ledu/columbia/gpsapp/MainActivity .sendMessage @ 181 :: False
BR::592 (0) :: 0 :: Ledu/columbia/gpsapp/MainActivity$Msg .toString @ 7 :: True
BR::592 (0) :: 0 :: Ledu/columbia/gpsapp/MainActivity$Msg .toString @ 317 :: False
BR::592 (0) :: 0 :: Ledu/columbia/gpsapp/MainActivity$Msg .toString @ 341 :: False
BR::592 (0) :: 0 :: Ledu/columbia/gpsapp/MainActivity$Msg .toString @ 351 :: False
